client/my-sites/site-indicator/site-indicator.jsx

```diff
@@ -417,9 +424,7 @@ export default connect(
 	},
 	{
 		updateWordPress,
-		trackSiteDisconnect: siteId => {
-			recordGoogleEvent( 'Jetpack', 'Clicked in site indicator to start Jetpack Disconnect flow' );
-			recordTracksEvent( 'calypso_jetpack_site_indicator_disconnect_start', { siteId } );
-		},
```

Comment:
I think you could use `composeAnalytics` here, see https://github.com/Automattic/wp-calypso/blob/master/client/state/analytics/README.md#api.

Reply:
Nice! Done. Noting that we don't need the `siteId` tracks prop—we get it as `blog_id` for free.